tests/test_main.py

```diff
@@ -1,31 +1,67 @@
+from curses import panel
+from urllib import response
+
+from app.main import app, knowledge_panel
 from fastapi.testclient import TestClient
-import requests
-from app.main import app
 
 client = TestClient(app)
 
-def test_answer_questions_brand_president():
-    response = client.get("/brand/president")
+
+def test_hello():
+    response = client.get("/")
     assert response.status_code == 200
-    assert response.json() == {"knowledge_panels": [
-                {
-                    "hunger-game": {
-                        "type": "hunger-game",
-                        "level": "questions",
-                        "elements": [
-                            {
+    assert response.json() == {
+        "message": "Hello from facets-knowledge-panels! Tip: open /docs for documentation"
+    }
+
+
+def test_knowledge_panel():
+    response = client.get("/knowledge_panel")
+    assert response.status_code == 422
+
+
+def test_knowledge_panel_badendpoint():
+    response = client.get("/knowledge_panel_bad")
+    assert response.status_code == 404
+
+
+def test_knowledge_panel_with_ctegory():
+    assert knowledge_panel(facet_name="category") == {
+        "knowledge_panels": [
+            {
+                "hunger-game": {
+                    "elements": [
+                        {
                             "element_type": "text",
                             "text_element": {
-                                "html":"<p><a href=\"https://hunger.openfoodfacts.org/?type=brand&value_tag=president\">Answer questions about brand president</a></p>\n"
+                                "html": "<p><a href='https://hunger.openfoodfacts.org/?type=category'></a></p>\n"
                             },
+                        }
+                    ]
+                }
+            }
+        ]
+    }
+
+
+def test_knowledge_panel_ctegory_with_country():
+    assert knowledge_panel(facet_name="category", country="India") == {
+        "knowledge_panels": [
+            {
+                "hunger-game": {
+                    "elements": [
+                        {
+                            "element_type": "text",
+                            "text_element": {
+                                "html": "<p><a href='https://hunger.openfoodfacts.org/?type=category&country=India'></a></p>\n"
                             },
-                        ],
-                    },
-                            
-                },
-            ],}
-
-def test_answer_questions_brand_president_bad():
-    response = client.get("/brand/bad_endpoint")
-    assert response.status_code == 404
-    assert response.json() == {'detail': 'Not Found'}
\ No newline at end of file
+                        }
+                    ]
+                }
+            }
+        ]
+    }
```

Comment:
As you already tested the hunger_game_kp function quite heavily, it's sufficient here to have a test including a facet_name, a facet_value and a country.

Note: the main usage will be facet_name + facet_value (not facet_name alone). So this is better to test this one.